content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -300,7 +270,7 @@ public abstract class Dog {
  
 With the above code snippet, we've created an abstract class and an abstract method. One pertinent feature of abstract class is that objects cannot be instantiated or created from it. But don’t forget, classes only exist to create an object. To be able to create an object from an abstract class. We must create a new class that inherits from the superclass(base, parent) class by using the extends keyword.
  
-Code snippet
+Code snippet to illustrate the `extends` keyword
```

Comment:
You need to make this a proper sentence and show add a colon at the end like so:

> Here is a code snippet to illustrate the `extends` keyword: